llvm/tools/sycl-post-link/ModuleSplitter.cpp

```diff
@@ -139,14 +139,14 @@ bool isESIMDFunction(const Function &F) {
 
 // This function makes one or two groups depending on kernel types (SYCL, ESIMD)
 EntryPointGroupVec
-groupEntryPointsByKernelType(const ModuleDesc &MD,
```

Comment:
`const` is removed in a number of places because some entry points can be changed now - their linkage fixed.